test/functional/ui/spell_spec.lua

```diff
@@ -0,0 +1,39 @@
+-- Test for scenarios involving 'spell'
+
+local helpers = require('test.functional.helpers')(after_each)
+local Screen = require('test.functional.ui.screen')
+local feed, insert, eq, eval, clear, expect = helpers.feed, helpers.insert, 
+  helpers.eq, helpers.eval, helpers.clear, helpers.feed_command, helpers.expect
+local Screen = require('test.functional.ui.screen')
+local feed_command = helpers.feed_command
+
+describe('combining long lines', function()
+  local screen
+  before_each(function()
+      clear()
+      screen = Screen.new(264, 4)
```

Comment:
Maybe it triggers if the screen is narrower like 60 columns

Reply:
I took that from ":set columns" on my screen, which is 272 and pared it down to not look so ugly on my screen when printed.  I also tested it with 30, 78, and 80 columns.  I get the same results.  I wonder if this just isn't the best way to trigger it.

Reply:
Could it be that the functional testing framework is just different enough from running the full application that this bug cannot occur in the tests?

Reply:
No, it runs full nvim instance, though some applicable option values might be different from `-u NONE`. I'll take a look.

Reply:
I got it to fail without the fix in place with 80 columns.  I'm not sure what is going on, but it consistently fails now.  Update imminent.